src/web/components/Approvals/ApprovalsTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { Participant } from '../../../api/entities/Participant';
+import { ApprovalItem } from './ApprovalItem';
+
+import './ApprovalsTable.scss';
+
+type ApprovalsTableProps = {
+  participants: Participant[];
+};
+
+function NoApprovals() {
+  return (
+    <div className='no-approvals-container'>
+      <img src='/group-icon.svg' alt='group-icon' />
```

Comment:
I guess we want a different icon here, but just putting it here as a placeholder